packages/engine/paima-sm/src/cde-erc721-transfer.ts

```diff
@@ -13,6 +19,8 @@ export default async function processErc721Datum(
   const { to, tokenId } = cdeDatum.payload;
   const toAddr = to.toLowerCase();
 
+  const isBurn = Boolean(toAddr.match(/^0x0+$/g));
```

Comment:
Checking the 0 address isn't enough. You also have to check `0x000000000000000000000000000000000000dEaD`